share/OpenMS/CHEMISTRY/Modomics.tsv

```diff
@@ -180,3 +180,7 @@ deoxycytidine	dC	dC	C			C9O4N3H13	227.0906	227.2176
 deoxyguanosine	dG	dG	G			C10O4N5H13	267.0968	267.2418
 deoxyuridine	dU	dU	U			C9O5N2H12	228.0746	228.2024
 thymidine	dT	dT	U			C10O5N2H14	242.0903	242.2290
+thio-adenosine	A*	A*	A	A	A	C10O4N5H13	267.0968	267.2449
```

Comment:
- are these names from modomics?
- where is the S? does it need to be in the formula?

Reply:
The names are not from modomics, and so if we keep them they should be moved to [Custom_RNA_modifications.tsv](https://github.com/OpenMS/OpenMS/blob/7185e2b4609fee952d60f90e7aae6b9a6c2cf091/share/OpenMS/CHEMISTRY/Custom_RNA_modifications.tsv).

The S is not part of the Nucleo**s**ide which is what populate the tsv's. The logic to add the phosphate linkages looks at the final character of the 'short name' and treats it as a thiol-linkage if that character is a star. 

The bigger question that I'm trying to decide is whether the NASequence parsing code should just populate the database with thiol versions of all of the entries in the tsvs or whether we should need to specify them. Any thoughts?